.github/workflows

```diff
@@ -0,0 +1,28 @@
+name: python-esiclient
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+          pip install -r requirements.txt
```

Comment:
I don't think you need to install requirements here: `tox` builds virtual environments and then installs the requirements inside there before running the tests (https://github.com/CCI-MOC/python-esiclient/blob/master/tox.ini#L18).